index.js

```diff
@@ -27,14 +27,13 @@ var formatOutput = function(report, file, options) {
     return err;
   });
 
-  var output = {
+  return {
```

Comment:
Removed the formatting, kept the mapping. This might be useful if people provide their own reporter-function